Gemfile

```diff
@@ -0,0 +1,61 @@
+source 'https://rubygems.org'
```

Comment:
Consider putting an explicit Ruby Version - `rvm` will pick up on this and auto-change when you `cd` into the folder